sdks/java/testing/tpcds/src/main/resources/queries/query1.sql

```diff
@@ -17,7 +17,7 @@
 with customer_total_return as
 (select sr_customer_sk as ctr_customer_sk
 ,sr_store_sk as ctr_store_sk
-,sum(SR_FEE) as ctr_total_return
```

Comment:
Are these syntax fixes based on some specific SQL dialect from the TPC-DS generation tool or do they come from another more 'Calcite' friendly dialect?

Reply:
These syntax fixes are based on Flink TPC-DS queries because Flink SQL is based on Apache Calcite.